scripts/crowsnest.sh

```diff
@@ -0,0 +1,229 @@
+    echo -e "Use this instance to setup your 'crowsnest.conf' and steering it's service.\n"
+    echo -e "Found the following instances:\n"
+    for i in ${1}; do
+        select_msg "${i}"
+    done
+    echo -e "\nLaunching crowsnest's configuration tool ..."
+    continue_config
+}
+
+# Helper funcs
+function clone_crowsnest(){
+    $(command -v git) clone "${CROWSNEST_REPO}" -b master "${CROWSNEST_DIR}"
+}
+
+function check_multi_instance(){
+    local -a instances
+    readarray -t instances < <(find "${HOME}" -regex "${HOME}/[a-zA-Z0-9_]+_data/*" -printf "%P\n" 2> /dev/null | sort)
+    if [[ "${#instances[@]}" -gt 1 ]]; then
+        status_msg "Multi Instance Install detected ..."
+        multi_instance_message "${instances[*]}"
+        if [[ -d "${HOME}/crowsnest" ]]; then
+            pushd "${HOME}/crowsnest" &> /dev/null || exit 1
+            if ! make config ;then
+                error_msg "Something went wrong! Please try again..."
+                if [[ -f "tools/.config" ]]; then
+                    rm -f tools/.config
+                fi
+                exit 1
+            fi
+            if [[ ! -f "tools/.config" ]]; then
+                log_error "failure while generating .config"
+                error_msg "Generating .config failed, installation aborted"
+                exit 1
+            fi
+            popd &> /dev/null || exit 1
+        fi
+    fi
+}
+
+function continue_config() {
+    local reply
+    while true; do
+        read -erp "Continue? [Y/n]: " -i "Y" reply
+        case "${reply}" in
+            [Yy]* )
+                break
+            ;;
+            [Nn]* )
+                warn_msg "Installation aborted by user ... Exiting!"
+                exit 1
+            ;;
+            * )
+                echo -e "\e[31mERROR: Please type Y or N !\e[0m"
+            ;;
+        esac
+    done
+    return 0
+}
+
+# function install_basic_deps(){
+#     local -a install
+#     local -a deps
+#     deps=(git make)
+#     for i in "${deps[@]}"; do
+#         if [[ -z "$(command -v "${i}")" ]]; then
+#             install+="${i}"
+#         fi
+#     done
+#     if [[ -n "${install[*]}" ]]; then
+#         ### Update system package info ( shameless stolen from klipper.sh )
+#         status_msg "Updating package lists..."
+#         if ! sudo apt-get update --allow-releaseinfo-change; then
+#             log_error "failure while updating package lists"
+#             error_msg "Updating package lists failed!"
+#             exit 1
+#         fi
+#         status_msg "Installing required packages..."
+#         if ! sudo apt-get install --yes "${install[@]}"; then
+#             log_error "failure while installing required crowsnest basic packages"
+#             error_msg "Installing required packages failed!"
+#             exit 1
+#         fi
+#     else
+#         ok_msg "All basic dependencies met, nothing to do!"
+#     fi
+# }
```

Comment:
Any specific reason as to why leave this here commented? Might as well just remove it, no?

Reply:
Well, obviously I was lost in my own code :rofl: 

No, will remove that asap

Reply:
Nevertheless, I tried your PR and it "worked fine on my machine"! 😁